main.css

```diff
@@ -173,3 +176,582 @@
 .headline .social-media-icons > li {
   list-style: none;
 }
+
+/* Works layout */
+.works {
+  display: flex;
```

Comment:
- [ ] It is required to use display grid in the works section as it is mentioned in the Project Requirements:
- In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.